### ISSUE TITLE
Incorrect s3 object replication status name

### ISSUE THREAD

**Issue by abaranovv** [aws/aws-sdk#242]
## Describe the bug
There is a wrong completed replication status name in the ReplicationStatus enum. According to [docs ](https://docs.aws.amazon.com/AmazonS3/latest/API/API_HeadObject.html#API_HeadObject_ResponseSyntax)name should be COMPLETED

## Expected Behavior
Status name should be `COMPLETED`

## Current Behavior
status is `COMPLETE`, so `ReplicationStatus.fromValue(String value)` returns `UNKNOWN_TO_SDK_VERSION`

## Steps to Reproduce
```
s3Client.headObject(HeadObjectRequest.builder()
                .bucket(bucketName)
                .key(key)
                .versionId(versionId)
                .build())
.replicationStatus();
```

## Possible Solution
fix status name in [service-2.json](https://github.com/aws/aws-sdk-java-v2/blob/master/services/s3/src/main/resources/codegen-resources/service-2.json#L8417)

## Your Environment

* AWS Java SDK version used: 2.13.49
* JDK version used: 11
* Operating System and version:

**Comment by rhowe** [comment]
Note that as a workaround you can (at least in Java SDKv2) call `replicationStatusAsString()` and compare to `"COMPLETED"`.

This bug made me question my own sanity

**Comment by debora-ito** [comment]
This was fixed upstream, `COMPLETED` was added to the ReplicationStatus enum -
https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/s3/model/ReplicationStatus.html

In Java SDK v2, the fix is part of SDK version `2.20.155` released on September 26, 2023. For the other SDKs, please check their changelog entries.
